### PR TITLE
Add setup script.

### PR DESCRIPTION
**Pull request by drapergeek**

### CODE REVIEW

File: lib/tasks/dev.rake [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    if Rails.env != "development"
+      raise "This task can only be create in the development environment"
+    end
+
+    DatabaseCleaner.strategy = :truncation
+    DatabaseCleaner.clean
+
+    puts "Creating development data..."
+    user = create(:user, email: 'user@example.com')
```
Comment (brentm5) [inline lib/tasks/dev.rake:17]: For now this looks fine but at some point we might want to make this another file or another rake task to run so someone can run this task alone.  Any thoughts?

Comment (drapergeek) [inline lib/tasks/dev.rake:17]: What do you mean? This is just a task for adding dev data that is called from the setup.sh thing.

Comment (brentm5) [inline lib/tasks/dev.rake:17]: HA... sorry read this wrong..... silly rabbit

Comment (brentm5) [inline lib/tasks/dev.rake:17]: :+1:

**Pull request merged**